releasenotes/notes/add-parameters-to-template-substitution-a1379cdbfcc10b5c.yaml

```diff
@@ -0,0 +1,88 @@
+---
+features:
+  - |
+    The Parameter class is now allowed in template optimization if the
+    parameterized gates are RZXGates and RXGates. It has been removed from 
+    the _remove_impossible() method for scenarios that are incompatible 
```

Comment:
We shouldn't talk about private implementations in the public release notes.

Reply:
Removed